File: HELLO_PYTHON/day06/mysql_select.py
# pip install mysql-connector-python

import mysql.connector

mydb = mysql.connector.connect(
    host='127.0.0.1',
    port='3305',
    user='root',
    password='python',
    database='python'
)

def select_all(): #조회 함수생성
    
    cur = mydb.cursor()                     #커서 객체생성
    sql ="SELECT * FROM emp"     #조회 SQL

    cur.execute(sql)                        #커서를 통한 SQL실행
    result = cur.fetchall()      #커서의 결과를 담는 객체

    for x in result:
        print(x)

select_all()                                #함수 실행

File: HELLO_NODE2/mysql_select.js
const mysql      = require('mysql'); //import 역할
const connection = mysql.createConnection({
  host     : 'localhost',
  user     : 'root',
  password : 'python',
  database : 'python',
  port : 3305 //기본 포트가 3306이므로 포트 지정도 해주어야 한다.
});

connection.connect();

connection.query('SELECT * from emp', (error, rows, fields) => {
//  if (error) throw error;
  console.log(rows);
  //console.log(error);
  //console.log(fields);
});

connection.end();


